modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -20,6 +20,12 @@ ifeval::[{suma-content} == true]
 endif::[]
 
 
+[IMPORTANT]
+====
+Traditional clients are not available on {centos}{nbsp}8.
+{centos}{nbsp}8 clients are only supported as Salt clients.
+====
+
```

Comment:
I am trying to add the channels for CentOS8 at https://github.com/SUSE/spacewalk/pull/10338. Will send the names on Monday, if @paususe decides he wants this into 4.0.4

Reply:
Channel for CentoS8:
```
centos8 centos8-appstream centos8-uyuni-client
```
BTW, we didn't mention channels for CentOS6, so maybe we should add the,
```
centos6 centos6-uyuni-client
```
And there's a typo at centos7, the client channel is duplicated.

Reply:
As strange as it sounds, it seems that centos8 doesn't have a separate updates repositories anymore.